docker-compose-localstack-tests.yml

```diff
@@ -15,6 +15,7 @@ services:
       - DOCKER_HOST=unix:///var/run/docker.sock
       - DEFAULT_REGION=eu-west-2
     volumes:
+      - "${TMPDIR:-/tmp/localstack}:/tmp/localstack"
       - "/var/run/docker.sock:/var/run/docker.sock"
```

Comment:
added this back, it is the easiest way to delete a corrupt ES database to have it on the host